docs/spark/tutorials/index.md

```diff
@@ -1,13 +1,16 @@
 ---
 title:  .NET for Apache Spark tutorials
 description: Explore the .NET for Apache Spark tutorials to learn how to integrate Apache Spark into your .NET applications.
-ms.date: 05/06/2019
+ms.date: 05/17/2019
 ---
 # .NET for Apache Spark tutorials
 
 The following tutorials enable you to understand how to use [.NET for Apache Spark](../index.yml) to build custom big data solutions with Apache Spark and integrate them into your .NET applications:
 
 - [Get started](get-started.md): demonstrates build and run a simple .NET for Apache Spark application.
+- [Deploy a .NET for Apache Spark application to Azure HDInsight](hdinsight-deployment.md): demonstrates how to deploy your .NET for Apache Spark application to Azure HDInsight.
+- [Deploy a .NET for Apache Spark application to Amazon EMR Spark](azure-emr-spark-deployment.md): demonstrates how to deploy your .NET for Apache Spark application to Amazon EMR Spark.
+- [Deploy a .NET for Apache Spark application to Azure HDInsight](databricks-deployment.md): demonstrates how to deploy your .NET for Apache Spark application to Databricks.
```

Comment:
The 4th bullet should be Databricks. It was a copy paste without an update.

Seems like the link + text is repetitive on all 3 bullets too.